src/main/java/in/org/projecteka/hiu/user/model/GatewayResponse.java

```diff
@@ -0,0 +1,14 @@
+package in.org.projecteka.hiu.user.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+
+@Builder
+@Getter
+@AllArgsConstructor
+@Data
+public class GatewayResponse {
+    private String requestId;
+}
```

Comment:
This model is already create somewhere. Please check.

Reply:
done